devops/azure-pipelines/azure-pipelines.yml

```diff
@@ -18,9 +17,24 @@ stages:
         parameters:
           jobName: 'Build'
           nodeVersion: '14.x'
+  - stage: Dockerize
+    jobs:
+      - template: ./dockerize-template.yml
+        parameters:
+          jobName: 'Dockerize'
+          containerRegistry: 'project-3-docker-hub'
+          repository: 'dajpearl/forge-frontend'
+          dockerPath: '**/Dockerfile'
+  - stage: Deploy
+    jobs:
+      - template: ./kubernetes-template.yml
+        parameters:
+          jobName: Deploy
+          serviceConnection: 'Alex-Dev-Cluster'
```

Comment:
why are you moving this cluster's config into the dev branch? the dev branch presumably is supposed to deploy to the dev environment as defined by terraform. if there's some serious delay in that, then i guess we could start with this, but there shouldn't be any delay; i last reviewed that terraform stuff like 3 days ago, and we've had the project a week now.